ground_truth_labeling_jobs/worker_contribution_private_workforce/worker_analysis.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>Provide available labeling job names. The training job fails with the following error:</p><p> </p><pre class="ql-syntax" spellcheck="false">* worker_analysis.ipynb ****************************************************************************
*
*    job name: papermill-worker-analysis-2021-03-19-10-14-28
*
*      kernel: Data Science
*
*      status: Failed     
*

---------------------------------------------------------------------------
Exception encountered at "In [5]":
---------------------------------------------------------------------------
ClientError                               Traceback (most recent call last)
&lt;ipython-input-5-f83f76783068&gt; in &lt;module&gt;
     10 # for each job download worker responses to a local folder
     11 for job in JOB_LIST:
---&gt; 12     download_worker_responses_for_job(job,work_dir)
     13 

&lt;ipython-input-5-f83f76783068&gt; in download_worker_responses_for_job(labelingJobName, directory)
      1 def download_worker_responses_for_job(labelingJobName=None,directory ="."):
----&gt; 2     response = sm_client.describe_labeling_job(LabelingJ

[...]

ClientError: An error occurred (ValidationException) when calling the DescribeLabelingJob operation: 1 validation error detected: Value '&lt;YOUR_LABELING_JOB_NAME_1&gt;' at 'labelingJobName' failed to satisfy constraint: Member must satisfy regular expression pattern: ^[a-zA-Z0-9](-*[a-zA-Z0-9]){0,62}
</pre><p> </p><p>I suggest you to change the list of strings to:</p><p> </p><p>JOB_LIST = [ 'your-labeling-job-name-1', 'your-labeling-job-name-2']</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2089/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
@mchoi8739 thanks for the feedback. The tricky part is that this will not fix the CI/CD issue. This notebook relies on an actual ground truth job being available in your account. However it is hard and maybe not cost effective to automate the ground truth labeling job. Please advice!